daemon/api_download.go

```diff
@@ -68,7 +68,7 @@ func streamOneSnap(c *Command, action snapDownloadAction, user *auth.UserState)
 	}}
 	snaps, err := getStore(c).SnapAction(context.TODO(), nil, actions, user, nil)
 	if err != nil {
-		return errToResponse(err, []string{action.SnapName}, InternalError, "cannot download snap: %v")
+		return errToResponse(err, []string{action.SnapName}, InternalError, "%v")
```

Comment:
why this change?

Reply:
We currently get the following error when a download fails:
```
error: cannot download snap: cannot download snap "fdfasd": snap not found
```
(note the duplication of "cannot download snap").

The store errors.go code already adds the "cannot download snap" prefix in SnapActionError.Error() (`"cannot %s snap %q: %v", op, name, e`). But given that this is cosmetic I can move it out of this PR and put it into a separate one given that its a bit distracting here.